.github/workflows/e2e-deploy.yml

```diff
@@ -93,58 +125,307 @@ jobs:
+            export HELM_EXTRA_OPTIONS="${HELM_EXTRA_OPTIONS} --set discoverer.image.tag=pr-${pr_num}"
+          fi
+          if curl -s "https://registry.hub.docker.com/v2/repositories/vdaas/vald-manager-compressor/tags/pr-${pr_num}" | jq '.name' | grep -v "null"; then
+            export HELM_EXTRA_OPTIONS="${HELM_EXTRA_OPTIONS} --set manager.compressor.image.tag=pr-${pr_num}"
+          fi
+          if curl -s "https://registry.hub.docker.com/v2/repositories/vdaas/vald-meta-cassandra/tags/pr-${pr_num}" | jq '.name' | grep -v "null"; then
+            export HELM_EXTRA_OPTIONS="${HELM_EXTRA_OPTIONS} --set meta.image.tag=pr-${pr_num}"
+          fi
+          if curl -s "https://registry.hub.docker.com/v2/repositories/vdaas/vald-manager-backup-cassandra/tags/pr-${pr_num}" | jq '.name' | grep -v "null"; then
+            export HELM_EXTRA_OPTIONS="${HELM_EXTRA_OPTIONS} --set manager.backup.image.tag=pr-${pr_num}"
+          fi
+          if curl -s "https://registry.hub.docker.com/v2/repositories/vdaas/vald-backup-gateway/tags/pr-${pr_num}" | jq '.name' | grep -v "null"; then
+            export HELM_EXTRA_OPTIONS="${HELM_EXTRA_OPTIONS} --set gateway.backup.image.tag=pr-${pr_num}"
+          fi
+          if curl -s "https://registry.hub.docker.com/v2/repositories/vdaas/vald-lb-gateway/tags/pr-${pr_num}" | jq '.name' | grep -v "null"; then
+            export HELM_EXTRA_OPTIONS="${HELM_EXTRA_OPTIONS} --set gateway.lb.image.tag=pr-${pr_num}"
+          fi
+          if curl -s "https://registry.hub.docker.com/v2/repositories/vdaas/vald-meta-gateway/tags/pr-${pr_num}" | jq '.name' | grep -v "null"; then
+            export HELM_EXTRA_OPTIONS="${HELM_EXTRA_OPTIONS} --set gateway.meta.image.tag=pr-${pr_num}"
+          fi
+          if curl -s "https://registry.hub.docker.com/v2/repositories/vdaas/vald-manager-index/tags/pr-${pr_num}" | jq '.name' | grep -v "null"; then
+            export HELM_EXTRA_OPTIONS="${HELM_EXTRA_OPTIONS} --set manager.index.image.tag=pr-${pr_num}"
+          fi
+          echo "HELM_EXTRA_OPTIONS=${HELM_EXTRA_OPTIONS}"
+          echo "HELM_EXTRA_OPTIONS=${HELM_EXTRA_OPTIONS}" >> $GITHUB_ENV
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libhdf5-dev
+      - name: Fetch Helm version
+        run: |
+          HELM_VERSION=`make version/helm`
+          echo "::set-output name=helm::${HELM_VERSION}"
+        id: version
+      - uses: rinx/setup-k3d@v0.0.2
+        with:
+          version: latest
+          name: vald
+          agents: 3
+      - name: check k3d
+        run: |
+          kubectl cluster-info
+      - uses: azure/setup-helm@v1
+        with:
+          version: ${{ steps.version.outputs.helm }}
+      - name: Helm version
+        run: |
+          helm version
+      - name: deploy vald
+        run: |
+          make k8s/external/scylla/deploy
```

Comment:
how about change members to 1. I think which accelarete the CI speed
https://github.com/vdaas/vald/blob/master/k8s/external/scylla/scyllacluster.yaml#L98

Reply:
Can I change the value? Is it okay?

Reply:
I think it's okay, if it works.
make scylla commands is for development we don't need to think so serious.
but it's better to overwrite by make ARGUMENT 

```
SCYLLA_CRD_DIR ?=k8s/external/scylla
k8s/external/scylla/deploy:
        kubectl apply -f $(SCYLLA_CRD_DIR)
```

Reply:
okay thanks!